### PR TITLE
Mailer

### PR DESCRIPTION
**Pull request by takeller**
## What's this PR do?
* Adds a mailer to send notifications when a request is made
* Pulls emails for all contributors to a campaign
## Where should the reviewer start? 
* src/mailers/mailer.py
## How should this be manually tested?
* Add a contributor with an email address you can check to a campaign OR
* Inside of src/views/RequestView change emails in line 32: `request_notification(emails, 'Test Link')`  to `[TempEmailAddress]`
* Using Postman send a post to create a new request for the campaign
* Check your email 
## Any background context you want to provide?
* Uses SendGrid, so this requires an API Key
* The email contains a dummy link, but will eventually point to the page where a contributor can vote on or view a request. 
## What are the relevant tickets?
*

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -4,12 +4,24 @@ class Development(object):
...
   DEBUG = True
   TESTING = False
   SQLALCHEMY_DATABASE_URI = os.getenv('SQLALCHEMY_DATABASE_URI')
   SQLALCHEMY_TRACK_MODIFICATIONS=False
+  MAIL_SERVER = 'smtp.sendgrid.net'
+  MAIL_PORT = 587
+  MAIL_USE_TLS = True
+  MAIL_USERNAME = 'apikey'
+  MAIL_PASSWORD = os.getenv('SENDGRID_API_KEY')
+  MAIL_DEFAULT_SENDER = "etho@ethoboost.com"
```
Comment (takeller) [inline src/config.py:24]: Config options specified by SendGrid

File: src/mailers/mailer.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import os 
+from sendgrid import SendGridAPIClient 
+from sendgrid.helpers.mail import Mail
+
+def request_notification(emails, request_link):
+    message = Mail(
+        from_email = 'ethoboost@gmail.com',
+        to_emails=emails, 
+        subject='Ethoboost Campaign Request',
+        html_content=f'There is a new request that requires your vote: {request_link}'
```
Comment (takeller) [inline src/mailers/mailer.py:10]: This is where the base message will live.

File: src/views/RequestView.py [hunk 3]
```diff
@@ -25,6 +25,12 @@ def create():
 
   request_info = Request(data)
   request_info.save()
+
+  # # Finds/sends emails. Needs update to use link for the request page. 
+  emails = [contributor.email for contributor in campaign.contributors]
+  if len(emails) > 0: 
+    request_notification(emails, 'Test Link')
```
Comment (takeller) [inline src/views/RequestView.py:32]: Eventually, I'd like to pass in the page where a contributor can view a request to this form. For now its just a dummy link.

**Review by AntonioJacksonII — APPROVED** [review]
Nice! This is a great feature to add.

**Review by perryr16 — APPROVED** [review]

**Pull request merged**